projects/cdk/schematics/ng-update/v4/steps/migrate-styles.ts

```diff
@@ -10,6 +11,7 @@ export function migrateStyles(): void {
             let fullText = sourceFile.getFullText();
 
             fullText = fullText
+                .replaceAll('textfield-placeholder()', 'label()')
```

Comment:
there is no `tui` prefix, unfortunately, seems like it is better to not migrate this

Reply:
fixed